asciidoc-confluence-publisher-converter/src/main/java/org/sahli/asciidoc/confluence/publisher/converter/AsciidocConfluencePage.java

```diff
@@ -154,30 +162,24 @@ private static String pageTitle(String pageContent) {
                 .orElseThrow(() -> new RuntimeException("top-level heading or title meta information must be set"));
     }
 
-    private static File parentFolder(Path pagePath) {
-        return pagePath.getParent().toFile();
-    }
-
-    private static Options options(String templateDir, File baseDir, String imagesOutDir) {
-        File templateDirFolder = new File(templateDir);
-
-        if (!templateDirFolder.exists()) {
+    private static Options options(Path templatesFolder, Path baseFolder, Path generatedAssetsTargetFolder) {
+        if (!(exists(templatesFolder))) {
             throw new RuntimeException("templateDir folder does not exist");
         }
 
-        if (!templateDirFolder.isDirectory()) {
+        if (!(isDirectory(templatesFolder))) {
```

Comment:
Why is this expression wrapped in parenthesis?

Reply:
I prefer to see the !-operator as a method/function call (i.e. `not(...)`) and therefore opt for wrapping the negated expression into parenthesis. That way, the !-operator is more visible (at least to me). 

Anyway, the majority of the usages in the code base do not use parenthesis, thus I removed the parenthesis here (and in all other usages) for consistency.

Fixed in e5535fba7fefdb14d003250e2b06a5887c708fca